iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/ExpressionImplTest.java

```diff
@@ -20,28 +20,33 @@
 package org.sonar.iac.arm.tree.impl.bicep;
 
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
-import org.sonar.iac.arm.parser.BicepParser;
 import org.sonar.iac.arm.parser.bicep.BicepLexicalGrammar;
+import org.sonar.iac.arm.parser.utils.Assertions;
 import org.sonar.iac.arm.tree.api.ArmTree;
 import org.sonar.iac.arm.tree.api.StringLiteral;
 import org.sonar.iac.arm.tree.api.bicep.SyntaxToken;
-import org.sonar.iac.common.extension.ParseException;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.assertj.core.api.Assertions.assertThatThrownBy;
-import static org.sonar.iac.common.testing.IacTestUtils.code;
 
-class ExpressionImplTest {
+class ExpressionImplTest extends BicepTreeModelTest {
 
-  BicepParser parser = BicepParser.create(BicepLexicalGrammar.EXPRESSION);
+  @Test
+  void shouldParseExpression() {
+    Assertions.assertThat(BicepLexicalGrammar.EXPRESSION)
```

Comment:
Typically we use a static import of assertions when possible.

Reply:
I agree with this, but this triggers a compilation error because of ambiguous method call:
```
assertThat (BicepLexicalGrammar) in Assertions and
assertThat (GrammarRuleKey) in Assertions match
```
A solution is to cast to `GrammarRuleKey`, but I didn't find this very elegant and so I moved toward the explicit class name, like we did for docker.
```
assertThat((GrammarRuleKey) BicepLexicalGrammar.EXPRESSION)
VS
Assertions.assertThat(BicepLexicalGrammar.EXPRESSION)
```
I would keep it like this at least for now, let me know if you prefer the other form I can handle it in a separate ticket.